aim/sdk/utils.py

```diff
@@ -1,7 +1,170 @@
 import os
 import pathlib
+import platform
+import shutil
+import subprocess
+import time
 import uuid
-from typing import Union
+from typing import Optional, Tuple, Union
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def get_username():
```

Comment:
what about moving sshfs utils to `aim/ext/sshfs` directory?

Reply:
yup, great advice, done ✅